Hangman/Hangman/main.swift

```diff
@@ -1,12 +1,139 @@
+
+//Alphabet library
+let alphabets: Set<Character> = Set("abcdefghijklmnopqrstuvwxyz")
+
+//Numbers array ** to protect against number entry**
```

Comment:
Great wait to account for different entries! Please remember to remove commented out code and comments before submitting your projects.